modules/rosetta/src/main/scala/org/tessellation/rosetta/server/model/network.scala

```diff
@@ -0,0 +1,52 @@
+package org.tessellation.rosetta.server.model
+
+import cats.Show
+import cats.implicits.toBifunctorOps
```

Comment:
please don't use `cats.implicits`. you should use `cats.syntax` as it's done in the whole project

Reply:
Changed the import to `cats.syntax.bifunctor._`.

Reply:
:+1: